src/main/kotlin/net/atos/zac/admin/ReferenceTableAdminService.kt

```diff
@@ -40,11 +38,14 @@ class ReferenceTableAdminService @Inject constructor(
 
     @Transactional(Transactional.TxType.REQUIRED)
     fun deleteReferenceTable(id: Long) {
-        val table = entityManager.find(ReferenceTable::class.java, id)
+        val referenceTable = entityManager.find(ReferenceTable::class.java, id)
+        require(!referenceTable.isSystemReferenceTable) {
+            "Deze referentietabel is een systeemtabel en kan niet verwijderd worden."
```

Comment:
I would expect the backend code to return a "code" of some sort...

Reply:
Ah you mean an error msg that can be translated by the frontend? Yes, that would be a lot nicer for sure. However, we have this all over the place currently and also this is a very exceptional exception as in: the frontend should make sure that this can never occur. So the only way to get this exception in the first place is with Postman etc. Still, I do agree.. Maybe another PR.. :-)